requirements-test.txt

```diff
@@ -1,5 +1,5 @@
 black==23.9.1
-flake8==6.0.0
+flake8==6.1.0
```

Comment:
I think we should swap out flake8 and isort for ruff.

Reply:
Agreed. But We can still merge this for now 👍